### PR TITLE
Update Angular to V9

### PR DESCRIPTION
**Pull request by van-fs**
This PR updates Angular to V9 and AngularFire to V6.  Also ran npm audit fix to auto fix.

### CODE REVIEW

File: tsconfig.app.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "src/polyfills.ts"
   ],
   "include": [
-    "src/**/*.ts"
+    "src/**/*.d.ts"
```
Comment (patrick-fs) [inline tsconfig.app.json:12]: I haven't seen this before, but likely stretching the limits of my `tsc` expertise. Why include the typedef files? Aren't these generated by `tsc`?

(FWIW, I see in tsconfig.json that `declaration` has been set to `false`)

**Review by patrick-fs — APPROVED** [review]
LGTM - I have a question about your tsconfigs

File: tsconfig.app.json (hunk 1 above)
Comment (van-fs) [reply tsconfig.app.json:12]: So `ng update` did this itself.  I'm going to assume Google knows more than I do, but I may dig into this more as I get a build/deploy going.

Comment (patrick-fs) [reply tsconfig.app.json:12]: SGTM. I'm down with "trust the tooling" :)

**Pull request merged**